src/assets/style/theme.js

```diff
@@ -0,0 +1,61 @@
+export const Breakpoints = {
+  xs: '640px',
+  sm: '830px',
+  md: '1100px',
+  lg: '1500px'
+}
+
+const Colors = {
```

Comment:
We can keep the primary, secondary, light, and dark if you implement them to match the current color scheme.